packages/cadl-ranch-specs/http/content-type/multipart-formdata/mockapi.ts

```diff
@@ -0,0 +1,30 @@
+import { passOnSuccess, ScenarioMockApi, mockapi } from "@azure-tools/cadl-ranch-api";
+import { resolvePath } from "@typespec/compiler";
+import { readFileSync } from "fs";
+import { fileURLToPath } from "url";
+
+export const Scenarios: Record<string, ScenarioMockApi> = {};
+
+const root = resolvePath(fileURLToPath(import.meta.url), "../../../../../");
+
+const pngFile = readFileSync(resolvePath(root, "assets/image.png"));
+const jpgFile = readFileSync(resolvePath(root, "assets/image.jpg"));
```

Comment:
Should we maybe make an helper/asset constant file so every spec that deals with binary doesn't have to reread the file and resolve the path.